cirq-google/cirq_google/engine/stream_manager.py

```diff
@@ -319,7 +313,8 @@ def _get_retry_request_or_raise(
     error: quantum.StreamError,
     current_request,
     create_program_and_job_request,
-    create_job_request: quantum.QuantumRunStreamRequest,
+    create_job_request,
```

Comment:
Can you keep the type hint here?

Reply:
It's still there, just moved down by 1 param

Reply:
that's not (typically) how type annotations work in python. They should be on each parameter